### ISSUE TITLE
jsona.serialize requires an obligatory includeNames

### ISSUE THREAD

**Issue by artem-galas** [olosegres/jsona#10]
`Jsona.serialize` requires an obligatory parameter - `includeNames`, although it should be optional.
I saw the test cases and I can't understand how they can work successfully. 
[(Test case)](https://github.com/olosegres/jsona/blob/master/tests/Jsona.test.ts#L32)
Here is the code below which causes this issue:
```ts
import Jsona from 'jsona/lib/Jsona';
const jsona = new Jsona();
const sessionData = {
      type: 'sessions',
      name: this.name,
      start_date_time: this.start_date_time
    };
const sessionDataSerialize = jsona.serialize({stuff: sessionData});
```

```
TS2345:Argument of type '{ stuff: { type: string; name: string; start_date_time: Date; }; }' is not assignable to parameter of type '{ stuff: TJsonaModel | TJsonaModel[]; includeNames: string[] | TJsonaNormalizedIncludeNamesTree; }'.
  Property 'includeNames' is missing in type '{ stuff: { type: string; name: string; start_date_time: Date; }; }'.
```

**Comment by olosegres** [comment]
You're right, this parameter conceived as optional, thanks for your PR!
Released in 1.1.3

**Comment by artem-galas** [comment]
BTW
I don't understand why tests was passed?
